pkg/cmd/operator/analyze.go

```diff
@@ -0,0 +1,85 @@
+package operator
+
+import(
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/scylladb/scylla-operator/pkg/genericclioptions"
+	"k8s.io/apimachinery/pkg/util/errors"
+)
+
+type AnalyzeOptions struct{
+	Mode string
+	GatherDirPath string
+}
+
+func (o *AnalyzeOptions) Validate() error {
```

Comment:
please move functions according to existing style. First Options struct definitions, then constructor (New*Options) then constructor for command and then Validate, Complete and Run at the end.